packaging/osx/clisdk/scripts/postinstall

```diff
@@ -9,10 +9,6 @@ INSTALL_DESTINATION=$2
 
 
 # A temporary fix for the permissions issue(s)
-chmod -R 755 $INSTALL_DESTINATION
-
-# Add the installation bin directory to the system-wide paths
-echo $INSTALL_DESTINATION | tee -a /etc/paths.d/dotnet
-
+chmod -R 755 $INSTALL_DESTINATIONs
```

Comment:
Is this a typo or is there supposed to be an "s" at the end of $INSTALL_DESTINATION?